tests/unit/components/Editor/ItemEditor.spec.js

```diff
@@ -189,6 +189,21 @@ describe("ItemEditor.vue", () => {
         .trigger("click");
       expect(wrapper.emitted()).toHaveProperty("delete-option");
     });
+
+    it("enables the survey mode in mcq", async () => {
+      await wrapper
+        .find('[data-test="surveyQuestionCheckbox"]')
+        .trigger("click");
+      expect(wrapper.find('[data-test="options"]').exists()).toBeTruthy();
+      expect(
+        wrapper.find('[data-test="subjectiveQuestionContainer"]').exists()
+      ).toBeFalsy();
```

Comment:
this never existed in the first place for mcq questions, right? So, why make this check?